src/addressResolvers/index.ts

```
@@ -0,0 +1,22 @@
+import { getAddress } from '@ethersproject/address';
+import * as ensResolver from './ens';
+import * as lensResolver from './lens';
+import * as unstoppableDomainsesolver from './unstoppableDomains';
+import { Address } from './utils';
+import cache from './cache';
+
+const RESOLVERS = [ensResolver, unstoppableDomainsesolver, lensResolver];
+
+export async function lookupAddresses(addresses: Address[]) {
+  const normalizedAddresses = addresses.slice(0, 250).map(a => getAddress(a));
```

Better to return an error if we pass more than `250` addresses

---

```suggestion
  const normalizedAddresses = addresses.slice(0, 250).map(getAddress);
```